apps/passport-server/src/services/issuanceService.ts

```diff
@@ -887,6 +893,25 @@ export class IssuanceService {
       return tickets;
     });
   }
+
+  /**
+   * Returns a promised verification of a PCD, either from the cache or,
+   * if there is no cache entry, from the multiprocess service.
+   */
+  private cachedVerify(serializedPCD: SerializedPCD): Promise<boolean> {
+    const key = JSON.stringify(serializedPCD);
+    const cached = this.verificationPromiseCache.get(key);
+    if (cached) {
+      return cached;
+    }
+    else {
```

Comment:
need to autoformat - the else should look like `} else {`

Reply:
Somehow needing to manually autoformat makes it seem like it's not really autoformatting

Reply:
#718

Reply:
Restarting VSCode and saving the file did indeed change the formatting though!